deploy/chart/templates/0000_50_18-service-monitor.yaml

```diff
@@ -0,0 +1,102 @@
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    name: olm-operator
+    namespace: {{ .Values.namespace }}
+    labels:
+      app: olm-operator
+  spec:
+    jobLabel: k8s-app
+    endpoints:
+    - port: http-metrics
+      interval: 30s
+      scheme: http
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    selector:
+      matchLabels:
+        app: olm-operator
+    namespaceSelector:
+      matchNames: {{ .Values.namespace }}
+---
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: olm-operator
+    namespace: {{ .Values.namespace }}
+    labels:
+      app: olm-operator
+  spec:
+    ports:
+    - name: http-metrics
+      port: 8081
+      protocol: TCP
+      targetPort: http
+    selector:
+      app: olm-operator
+---
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    name: catalog-operator
+    namespace: {{ .Values.namespace }}
+    labels:
+      app: catalog-operator
+  spec:
+    jobLabel: k8s-app
+    endpoints:
+    - port: http-metrics2
+      interval: 30s
+      scheme: http
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    selector:
+      matchLabels:
+        app: catalog-operator
+    namespaceSelector:
+      matchNames: {{ .Values.namespace }}
+---
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: catalog-operator
+    namespace: {{ .Values.namespace }}
+    labels:
+      app: catalog-operator
+  spec:
+    ports:
+    - name: http-metrics2
+      port: 8081
+      protocol: TCP
+      targetPort: http
+    selector:
+      app: catalog-operator
+---
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: operator-lifecycle-manager-metrics
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: operator-lifecycle-manager-metrics
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: {{ .Values.namespace }}
+---
```

Comment:
Is something missing here? I don't see the `ClusterRole` that the CRB binds to, and I don't see anything binding to the role. I assume `prometheus-k8s` serviceaccount already exists?

Reply:
I'm really not sure what this service account should be, but I was assuming this service account existed.

Reply:
Where did the `ClusterRoleBinding` and `Role` come from? Are there docs on integrating with cluster-monitoring? I would have expected rbac pairs of (role, rolebinding), (clusterrole, clusterrolebinding), or (clusterrole, rolebinding), but not (role, clusterrolebinding) - it's not clear to me how these would be used

Reply:
There's an internal doc (sorry if anybody else is reading along): https://docs.google.com/document/d/1rCKAYTrYMESjJDyJ0KvNap05NNukmNXVVi6MShISnOw/edit#. I didn't mean to mix cluster/role pairs. I was basing the RBAC permissions off of https://github.com/openshift/cluster-openshift-controller-manager-operator/tree/master/manifests (and didn't see a service account created there).